src/index.html

```diff
@@ -1,13 +1,333 @@
+          <a href="tel: +1 234 5555-55-55" class="menu__order">Call to order</a>
+        </div>
+      </div>
+    </nav>
+
+    <main>
+      <section class="page__section" id="about-us">
+        <div class="container">
+          <h2 class="page__section-title">
+            Recommended
+            <div class="product">
+              <div class="grid">
+                <div class="product__item-1 grid__item--1-4 grid__item--on744px-2-5 grid__item--on320px-1-2">
+                  <img
+                    src="./images/product_1.png"
+                    alt="product_1"
+                    class="product__photo"
+                  >
+                  <div class="product__description">
+                    <h3 class="product__title">
+                      Bose portable Smart speaker
+                    </h3>
+                    <div class="product__type">
+                      Smart home
+                    </div>
+                    <div class="product__price">
+                      $ 399.00
+                    </div>
+                  </div>
+                </div>
+
+                <div class="product__item-2 grid__item--5-8 grid__item--on744px-2-5 grid__item--on320px-1-2">
+                  <img
+                    src="./images/product_2.png"
+                    alt="product_2"
+                    class="product__photo"
+                  >
+                  <div class="product__description">
+                    <h3 class="product__title">
+                      SoundLink Flex Bluetooth speaker
+                    </h3>
+                    <div class="product__type">
+                      Portable bluetooth
+                    </div>
+                    <div class="product__price">
+                      $ 149.00
+                    </div>
+                  </div>
+                </div>
+
+                <div class="product__item-3 grid__item--9-12 grid__item--on744px-2-5 grid__item--on320px-1-2">
+                  <img
+                    src="./images/product_3.png"
+                    alt="product_3"
+                    class="product__photo"
+                  >
+                  <div class="product__description">
+                    <h3 class="product__title">
+                      SoundLink Color Bluetooth speaker II
+                    </h3>
+                    <div class="product__type">
+                      Portable bluetooth
+                    </div>
+                    <div class="product__price">
+                      $ 129.00
+                    </div>
+                  </div>
+                </div>
+              </div>
+            </div>
+          </h2>
+        </div>
+      </section>
+
+      <section class="page__section" id="gallery">
+        <div class="container">
+          <h2 class="page__section-title">
+            Browse Bose
+            <br>
+            products by category
+          </h2>
+
+          <div class="product product--flex">
```

Comment:
one more `product` section?
if this is a part of the previous section so add it there!